tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2617,6 +2640,12 @@ route/test_static_route.py::test_static_route_ecmp_ipv6:
 #######################################
 #####              sflow          #####
 #######################################
+sflow:
```

Comment:
is this an asic limitation?

Reply:
> is this an asic limitation?

No, this is an unsupported features according to qual doc.